### ISSUE TITLE
Selecting light types and molecules in Jaws can use screen reader, rather than our, focus

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/molecules-and-light#259]
**Test device**
Dell
**Operating System**
Win 10
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/QA/issues/461. Found using Jaws.
When trying to select a type of light or molecule Jaws will often use its light green highlight instead of the normal pink highlight. This can even occur when you used tab to get to that section. If you press tab while doing this, the pink highlight appears, but attempting to select something using normal keyboard commands after this will not move the pink highlight, but instead move the green one. It seems to do this if you were, before attempting to make a selection, either in the control panel or listening to the scene description. To fix this you need to move the light intensity slider and then use Tab to get to the selection you want to use.
**Steps to reproduce**
1. Use Jaw to listen to the scene description until it reaches the light type selection
2. Try to select different light and molecule types
3. Navigate back to the light slider and use it
4. Tab back to the light or molecule type menus and use them

**Visuals**
Issue:
https://drive.google.com/file/d/1jddMBMXWN4T3XWybhfT5V3lZveBD0Y1b/view?usp=sharing
How to Fix:
https://drive.google.com/file/d/1klxLuMbcIgmVcNhxedEZv8FK8ZOnyxEJ/view?usp=sharing

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Molecules and Light‬
URL: https://phet-dev.colorado.edu/html/molecules-and-light/1.5.0-dev.21/phet/molecules-and-light_en_phet.html?ea&a11y&supportsSound
Version: 1.5.0-dev.21 2019-12-13 19:43:51 UTC
Features missing: touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/79.0.3945.79 Safari/537.36
Language: en-US
Window: 1536x722
Pixel Ratio: 2.5/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by jessegreenberg** [comment]
Thanks for pointing out @KatieWoe. This is actually an expected difference introduced by JAWS. JAWS removes the "auto selection" of arrow keys and by default lets the user read what the radio button is before selecting it. If the user presses spacebar or enter while the green highlight is over a button that is not selected, the new radio button still gets pressed correctly and gets surrounded by our pink highlight.

Unfortunately, JAWS does not make it possible for our pink highlight to follow the green highlight when "auto selection" is disabled.

One more quirk: If you press "enter" on a radio button it will turn on "auto selection" and from there you can use the arrow keys like normal.

Back to @KatieWoe to review and close if this answers the issue.

**Comment by KatieWoe** [comment]
For the most part. My only remaining concern is https://github.com/phetsims/molecules-and-light/issues/260, so this can be closed.

**Comment by jessegreenberg** [comment]
OK soundsd good, closing this one. #260 may still be an issue, we will continue there.
